services/src/main/java/org/exoplatform/addons/gamification/service/mapper/DomainMapper.java

```diff
@@ -38,15 +41,14 @@ public DomainEntity domainDTOToDomain(DomainDTO domainDTO) {
                 domain.setCreatedBy(domainDTO.getCreatedBy());
                 domain.setLastModifiedBy(domainDTO.getLastModifiedBy());
                 if (domainDTO.getLastModifiedDate() != null) {
+                    try {
                     domain.setLastModifiedDate(formatter.parse(domainDTO.getLastModifiedDate()));
+                    }catch (ParseException e) {
+                        LOG.warn("Connot parse domin {} last modified date", domain.getId());
```

Comment:
* Please log the exception 
* there is a typo on ``Connot``
* please check the indentation

Reply:
Ok